src/Orchard.Cms.Web/Startup.cs

```diff
@@ -47,4 +49,34 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
             app.UseModules();
         }
     }
+
+    public class DesignTimeMvcBuilderConfiguration : IDesignTimeMvcBuilderConfiguration
+    {
+        public void ConfigureMvc(IMvcBuilder builder)
+        {
+            var serviceProvider = builder.Services.BuildServiceProvider();
+            var env = builder.Services.BuildServiceProvider().GetRequiredService<IHostingEnvironment>();
```

Comment:
`var env = serviceProvider.GetRequiredService...` since you've already created a container.